### PR TITLE
Include MMTF-Cpp in the superbuild

### PR DESCRIPTION
**Pull request by frodofine**
Subject explains it all! I'm including mmtf-cpp as a required dependency, but I'm willing to change this if needed.

Signed-off-by: Jonathan Fine <finej@purdue.edu>

### CODE REVIEW

File: cmake/External_mmtfcpp.cmake [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Note that Open Babel is GPLv2 only licensed, one particular problem this
```
Comment (cryos) [inline cmake/External_mmtfcpp.cmake]: It isn't clear if these comments are relevant

File: cmake/External_msgpackc.cmake [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# An external project for spglib
```
Comment (cryos) [inline cmake/External_msgpackc.cmake]: Same here, copy/paste left more than intended?

**Comment by cryos** [comment]
Super minor questions on comments, this built for me, and seemed to work in my initial testing. Thanks for moving this to the superbuild!

**Comment by frodofine** [comment]
Yes, those are copy-paste mistakes! I've pushed a fix.

**Pull request merged**